### PR TITLE
Adds ability to configure account metrics via tcld commands

### PR DESCRIPTION
**Pull request by mastermanu**

### CODE REVIEW

File: app/account.go [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+						{
+							Name:  "enable",
+							Usage: "Enables the metrics endpoint. CA Certificates *must* be configured prior to enabling the endpoint",
+							Action: func(ctx *cli.Context) error {
+								a, err := c.getAccount()
+								if err != nil {
+									return err
+								}
+
+								if a.Spec.Metrics != nil && a.Spec.Metrics.EnableMetricsEndpoint {
```
Comment (mastermanu) [inline app/account.go:119]: Will change this proto to just be "enable"

File: app/account.go [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+									return errors.New("metrics endpoint is already enabled")
+								}
+
+								if a.Spec.Metrics == nil || a.Spec.Metrics.AcceptedMetricsClientCa == "" {
```
Comment (mastermanu) [inline app/account.go:123]: will change this proto to just AcceptedClientCa

File: protogen/api/auditlog/v1/message.pb.go [hunk 3]
```diff
@@ -0,0 +1,1736 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: api/auditlog/v1/message.proto
+
+package auditlog
```
Comment (underrun) [inline protogen/api/auditlog/v1/message.pb.go:4]: why is the auditlog proto go file added as part of this?

Comment (mastermanu) [reply protogen/api/auditlog/v1/message.pb.go:4]: because it came with syncing to the latest protos

**Review by underrun — APPROVED** [review]

**Pull request merged**